### PR TITLE
fix: disable projects

### PR DESCRIPTION
**Pull request by FredrikOseberg**

### CODE REVIEW

**Review by chriswk — APPROVED** [review]
Nice. Removes a way our users could get into a strange state. :)

File: src/test/e2e/api/client/feature.env.disabled.e2e.test.ts [hunk 1]
```diff
@@ -51,20 +51,3 @@ test('returns feature toggle for default env', async () => {
             expect(res.body.features[0].strategies).toHaveLength(1);
         });
 });
-
-test('returns feature toggle for default env even if it is removed from project', async () => {
```
Comment (FredrikOseberg) [inline src/test/e2e/api/client/feature.env.disabled.e2e.test.ts:55]: This test doesn't make sense anymore when you can't remove all environments from a project. Is there another way we could test this @chriswk @ivarconr ?

Comment (ivarconr) [reply src/test/e2e/api/client/feature.env.disabled.e2e.test.ts:55]: You could always have the test going directly to the store and bypassing. Could be good to make sure that the API will still handle data even if the environment is removed. The use case is that the toggle should still exits for clients even if it does not have any active environments

Comment (FredrikOseberg) [reply src/test/e2e/api/client/feature.env.disabled.e2e.test.ts:55]: Great! I'll do that.

**Review by ivarconr — APPROVED** [review]

**Pull request merged**